doc/release-notes.md

```diff
@@ -28,22 +35,378 @@ reindex or re-sync the whole chain.
+
+Descriptor Wallets move to a per-wallet watchonly model. Instead an entire wallet is considered to be
+watchonly depending on whether it was created with private keys disabled. This eliminates the need
+to distinguish between things that are watchonly and things that are not within a wallet itself.
+
+This change does have a caveat. If a Descriptor Wallet with private keys *enabled* has
+a multiple key descriptor without all of the private keys (e.g. `multi(...)` with only one private key),
+then the wallet will fail to sign and broadcast transactions. Such wallets would need to use the PSBT
+workflow but the typical GUI Send, `sendtoaddress`, etc. workflows would still be available, just
+non-functional.
+
+This issue is worsened if the wallet contains both single key (e.g. `pkh(...)`) descriptors and such
+multiple key descriptors as some transactions could be signed and broadast and others not. This is
+due to some transactions containing only single key inputs, while others would contain both single
+key and multiple key inputs, depending on which are available and how the coin selection algorithm
+selects inputs. However this is not considered to be a supported use case; multisigs
+should be in their own wallets which do not already have descriptors. Although users cannot export
+descriptors with private keys for now as explained earlier.
+
+Configuration
+-------------
+
+### New cmd-line options
+- `-networkactive=` Enable all P2P network activity (default: 1). Can be changed by the setnetworkactive RPC command"
+- A new configuration flag `-maxapsfee` has been added, which sets the max allowed
+  avoid partial spends (APS) fee. It defaults to 0 (i.e. fee is the same with
+  and without APS). Setting it to -1 will disable APS, unless `-avoidpartialspends`
+  is set. (dash#5930)
+- A new CLI `-addrinfo` command returns the number of addresses known to the
+  node per network type (including Tor v2 versus v3) and total. This can be
+  useful to see if the node knows enough addresses in a network to use options
+  like `-onlynet=<network>` or to upgrade to current and future Tor releases
+  that support Tor v3 addresses only.  (#5904)
+
+### Updated cmd-line options
+- The `-debug=db` logging category, which was deprecated in v0.18 and replaced by
+  `-debug=walletdb` to distinguish it from `coindb`, has been removed. (#6033)
+- The `-banscore` configuration option, which modified the default threshold for
+  disconnecting and discouraging misbehaving peers, has been removed as part of
+  changes in this release to the handling of misbehaving peers. Refer to the
+  section, "Changes regarding misbehaving peers", for details. (dash#5511)
+- If `-proxy=` is given together with `-noonion` then the provided proxy will
+  not be set as a proxy for reaching the Tor network. So it will not be
+  possible to open manual connections to the Tor network for example with the
+  `addnode` RPC. To mimic the old behavior use `-proxy=` together with
```

Comment:
```suggestion
- If `-proxy=` is given together with `-noonion` then the provided proxy will
  not be set as a proxy for reaching the Tor network. So it will not be
  possible to open manual connections to the Tor network, for example, with the
  `addnode` RPC. To mimic the old behavior use `-proxy=` together with
```